runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsConfig.java

```diff
@@ -58,24 +58,35 @@ default boolean lvl()
 	}
 
 	@ConfigItem(
-		position = 2,
-		keyName = "clue",
-		name = "Clue Command",
-		description = "Configures whether the Clue command is enabled"
+			position = 2,
```

Comment:
What's up with the extra indents here

Reply:
Did this via the GitHub online editor and I didn't notice the extra spaces. I'll configure my IDE to account for these rules next time....

Reply:
https://github.com/runelite/runelite/wiki/Code-Conventions you can import XML mentioned here to IntelliJ to have correct ones

Reply:
Thanks, appreciate it.